pkgs/by-name/fi/field-monitor/package.nix

```diff
@@ -0,0 +1,106 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  cargo,
+  meson,
+  ninja,
+  rustPlatform,
+  rustc,
+  pkg-config,
+  glib,
+  gsettings-desktop-schemas,
+  gtk4,
+  libadwaita,
+  libvirt,
+  gst_all_1,
+  desktop-file-utils,
+  appstream,
+  appstream-glib,
+  wrapGAppsHook4,
+  xdg-desktop-portal,
+  blueprint-compiler,
+  libxml2,
+  freerdp,
+  spice-protocol,
+  spice-gtk,
+  vte-gtk4,
+  gtk-vnc,
+  usbredir,
+  libepoxy,
+  libGL,
+  openssl,
+}:
+stdenv.mkDerivation rec {
+  pname = "field-monitor";
+  version = "48.0";
+
+  src = fetchFromGitHub {
+    owner = "theCapypara";
+    repo = "field-monitor";
+    tag = "v" + version;
+    hash = "sha256-ma1ltI2sVNVpSoFd5H5Tq7fo/uepX1OzM1qIKvMFs8c=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoVendor {
+    inherit src;
+    hash = "sha256-3FimnzO9ja0OLaPRrMipVePD6x6Di5VvZRG31V8NSfg=";
+  };
+
+  mesonBuildType = "release";
+
+  postInstall = ''
+    wrapProgram $out/bin/de.capypara.FieldMonitor --prefix PATH ':' "$out/libexec"
+  '';
+
+  nativeBuildInputs = [
+    glib
+    gtk4
```

Comment:
You don't need to duplicate `buildInputs` over `nativeBuildInputs`. Can you remove the duplicated entries?